05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,673 @@
+  as_survey_rep(type = "BRR", repweights = starts_with("rep"),
+                combined_weights = FALSE, weight=wt)
+
+des_brr_scd
+
+summary(des_brr_scd)
+```
+
+Note that `combined_weights` was specified as `FALSE` because these weights are simply specified as 0 and 2 and do not incorporate the overall weight. When printing the object, the type of replication is noted as Balanced Repeated Replicates, the replicate weights are specified, and the weight variable. When looking at the summary, the only additional information provided are the variables included.
+
+### Fay's BRR Method
+
+Fay's BRR method for replicate weights still uses a Hadamard matrix to construct replicate weights but rather than deleting PSUs for each replicate, half of the PSUs have a replicate weight which is the main weight multiplied by $\rho$ and the other half have the main weight multiplied by $(2-\rho)$ where $0 \le \rho < 1$. Note that when $\rho=0$, this is equivalent to the standard BRR weights. 
+
+#### The math {-}
+
+The standard error estimate for $\hat{\theta}$ is slightly different and calculated as:
+
+$$se(\hat{\theta})=\sqrt{\frac{1}{R (1-\rho)^2} \sum_{r=1}^R \left( \hat{\theta}_r-\hat{\theta}\right)^2}$$
+
+#### The syntax {-} 
+
+The syntax is very similar for BRR and Fay's BRR. If a dataset had WT0 for the main weight and had 20 BRR weights indicated WT1, WT2, ..., WT20, and Fay's multiplier is 0.5, use the following syntax:
+
+```r
+des_fay <- dat %>%
+  as_survey_rep(weights = WT0, repweights= num_range("WT", 1:20),
+                type="Fay", mse=TRUE, rho=0.5)
+```
+
+#### Example {-} 
+
+The 2015 RECS uses Fay's BRR weights with the final weight as NWEIGHT and replicate weights as BRRWT1 - BRRWT96 with $\rho=0.5$^[Using the 2015 microdata file to compute estimates and standard errors (RSEs):  https://www.eia.gov/consumption/residential/data/2015/pdf/microdata_v3.pdf]. On the file, DOEID is a unique identifier for each respondent, TOTALDOL is the total cost of energy, TOTSQFT_EN is the total square footage of the residence, and REGOINC is the Census region.
+
+To specify this design, use the following syntax:
+
+```{r}
+#| label: recs
+#| eval: TRUE
+#| cache: TRUE
+recs_in <- read_csv(here::here("RawData", "RECS_2015", "recs2015_public_v4.csv"))
+
+des_recs <- recs_in %>%
+  as_survey_rep(weights=NWEIGHT, repweights=BRRWT1:BRRWT96, type="Fay",
+                rho=0.5, mse=TRUE, variables=c(DOEID, TOTALDOL, TOTSQFT_EN, REGIONC))
+
+des_recs
+
+summary(des_recs)
+```
+
+In specifying the design, the `variables` option was also used to include which variables might be used in analyses. This is optional but can make your object smaller. When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Fay's variance method (rho= 0.5 ) with 96 replicates and MSE variances.` and the variables are included. No weight or probability summary is included as was done in some other design objects.
```

Comment:
```suggestion
In specifying the design, the `variables` option was also used to include which variables might be used in analyses. This is optional but can make your object smaller. When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Fay's variance method (rho= 0.5) with 96 replicates and MSE variances` and the variables are included. No weight or probability summary is included as was done in some other design objects.
```

Reply:
I know you are copy/pasting, but it looks very weird in the rendered book.  Especially with the period in the middle of the sentence. I recommend making these small edits to make it more readable.